test/functionalTest/cases/2584_service_path_header_in_notifs/service_path_header_in_notifs.test

```diff
@@ -0,0 +1,196 @@
+# Copyright 2017 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
> A new one is both faster and better suited for the problem

Faster and better in the context of a given isolated problem solution task, but not so fast and good in the overall. Each new .test is another lilttle piece of stuff to mantain (and when you have to do a large refactor effort, e.g. XML removal, these little pieces, all together, have a significative cost). In addition, each new .test is a little more delay in make test, so impacting our development speed (even in the case of "I'm working on the next development while make test ends" part of your mind is still in the PR that you haven't merged yet due to the make test... at least in my case I cannot do complete context change as computers do ;).

It isnt' a golde rule about this, but something to think about (food for thought).